### DISCUSSION TITLE
Laser cutting module changed but not cutting

### DISCUSSION THREAD

**Discussion by coejoa** [arkypita/LaserGRBL#1411] [Q&A]
I had bought myself a genmitsu le5040 laser engraver and thought it will cut through plywood of 3mm, but it did not as there where misunderstanding or communication between the seller and myself and the output of 5.5w.

Last week I bought a Neje 30w output 7.5w laser module and replace the module with now problem

I need help on lasergrbl settings - in my book icon it does not show much of laser modules cause mine is now (30w) 7.5w and it shows only orthur 20w (5.5w) and other

Do I need other settings or software to be able to cut

![Capture](https://user-images.githubusercontent.com/82512211/114703468-9e4b0700-9d25-11eb-8480-96f4838355eb.JPG)

![20210414_133124](https://user-images.githubusercontent.com/82512211/114703940-452fa300-9d26-11eb-8a0e-4e2de3f9c075.jpg)

![20210414_133142](https://user-images.githubusercontent.com/82512211/114704012-5e385400-9d26-11eb-99b6-69f07f487b4f.jpg)

**Comment by gmmanonymus111** [comment]
Since NEJE's 30W and 40W lasers are fairly new (and expensive), there were no test as which settings should be used for which material. That is why Material DB lacks those values. Sadly, you need to test it out what power/speed combination works for best. I'd go with `100%` power and `150-300mm/min`, `1 pass`. See how it turns out, then you can fine-tune those settings, and add more passes if needed.

**Reply by coejoa** [reply]
Thank you very much
what do you think would be the best optimal focus distance between the laser and material

**Reply by gmmanonymus111** [reply]
For cutting, I think the best is as close as possible, about 2.5-7.5 cm (but watch out for the smoke, as it can make the lens dirty, reducing your lasers efficiency)

**Reply by coejoa** [reply]
But even it's 30W the output are 7.5W 
I did try 150 and 300 with full power no luck

**Reply by gmmanonymus111** [reply]
Try with multiple passes, sometimes plywood need 2-3 passes to be cut. Also, is it has a lot of glue/adhesive, your laser won't cut it.

**Reply by coejoa** [reply]
Will do thank you so what I can learn from this exercise don't trust what they sell to you


![Untitled](https://user-images.githubusercontent.com/82512211/114712279-36e68480-9d30-11eb-8069-3ce18bb770d5.png)

**Reply by coejoa** [reply]
Just another question the "select your configuration"  lasergrbl settings - in my book icon will it have an effect on what I choose even if my 30w (7w) are not in the list

**Reply by gmmanonymus111** [reply]
Well, those are basically just shortcuts for speed and laser power settings. If you want, you could try to use the 20W (5.5W) ones, with slightly more speed or less power %.
